specification/frontdoor/resource-manager/Microsoft.Network/stable/2021-06-09/webapplicationfirewall.json

```diff
@@ -0,0 +1,1038 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-06-09",
+    "title": "WebApplicationFirewallManagement",
```

Comment:
I don't think a new API version is strictly required since these changes are limited to read-only APIs. This used to be the case, but it's a little less clear in the latest breaking change policy.

[https://microsoft.sharepoint.com/teams/azureecosystem/servicerings/Shared%20Documents/Breaking%20Changes/Azure%20Breaking%20Changes%20Program%20Definition-%20V1.2.pdf](https://microsoft.sharepoint.com/teams/azureecosystem/servicerings/Shared%20Documents/Breaking%20Changes/Azure%20Breaking%20Changes%20Program%20Definition-%20V1.2.pdf)